Announce the launch of Skolar for the scikit-learn community (#217)
scikit-learn/blog: _posts/2025-06-30-probabl-skolar.md

---
#### Blog Post Template ####

#### Post Information ####
title: "Skolar: an open-source initiative to democratize open data science"
date: June 30, 2025

#### Post Category and Tags ####
# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
categories:
  - Updates
tags:
  - Sponsor
  - Open Source
  - Machine Learning

#### Featured Image ####
featured-image: BSD_watermark.svg

#### Author Info ####
# Can accomodate multiple authors
# Add SQUARE Author Image to /assets/images/author_images/ folder
postauthors:
  - name: Skolar
    website: https://skolar.probabl.ai/
    image: "skolar-logo.png"
  - name: Pénélope Gittos
    website: https://www.linkedin.com/in/gittospenelope-data-analyst-growth-bilingual/
    image: "penelope_gittos.jpeg"
---
<div>
  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
  {% include postauthor.html %}
</div>

<span style="color:red">*This blog post has been submitted by Probabl, a sponsor of scikit-learn.* </span>

> The scikit-learn project always puts efforts on education to build and nurture a

Reviewer: A few suggestions, even if they say something slightly different to my interpretation of the original
```suggestion
The scikit-learn project values educational efforts that build and nurture a
```

```suggestion
The scikit-learn project has always put effort into education in order to build and nurture a
```

```suggestion
The scikit-learn project has always put a lot of effort into education in order to build and nurture a
```

(I don't know if posting three suggestions will work or not, maybe don't use the GitHub UI to apply it?)

> strong vibrant open-source community. The goal is straightforward: give

Reviewer: ```suggestion
strong vibrant open-source community. The goal of this is straightforward: give
```

everyone, everywhere, the tools they need to easily grasp, engage with, and
meaningfully contribute to data science using open-source software. This mission
is shared and actively supported by [Probabl](https://probabl.ai/), a company
that helps maintain scikit-learn by employing many of its core contributors and
investing in its long-term sustainability. With their support and a deep

> commitment from the community, we continue building bridges between research,

Reviewer: "... With their support ... , we continue"

Maybe we can make it more explicit who is who here. At least I'm a bit lost if for example the "their" is the group of engineers employed by probabl, or probal or someone else? If it is probabl I think "With probabl's support and ..." would be clearer

> investing in its long-term sustainability. With their support and a deep
> commitment from the community, we continue building bridges between research,

Reviewer 2: ```suggestion
investing in its long-term sustainability. With Probabl's support and a deep
commitment from the community, the scikit-learn ecosystem continues building bridges between research,
```

software, and education.

When the [Inria scikit-learn MOOC](https://inria.github.io/scikit-learn-mooc/)

> (Massive Open Online Course) first went live, our community got a front-row seat

Reviewer: ```suggestion
(Massive Open Online Course) first went live, the community got a front-row seat
```

```suggestion
(Massive Open Online Course) first went live, the scikit-learn community got a front-row seat
```

Again I think it is a bit unclear whose community it is if we use "our". Given the post is written by probabl the most straightforward interpretation is that "our community" is "probabl's community" which doesn't seem quite right. I like "the community", avoids any suggestion that it belongs to anyone

open-source educational initiative, built directly from your feedback and all
the lessons we've learned along the way. Developed and extended by those same
core developers of scikit-learn, Skolar is designed specifically for data
science practitioners, offering hands-on, high-quality learning resources
grounded in real-world applications and open-source values.

Skolar exists to boost our shared values: openness, teamwork, and practicality.
It offers clear, interactive tutorials and structured courses carefully designed
to match industry challenges and specialized use-cases. But even more
importantly, it captures the true spirit of open source: encouraging
collaboration, peer-to-peer learning, and guidance from experts.

Right now, we’re just at the beginning. Today, you can dive into our
Scikit-learn Associate Practitioner online course, adapted from the popular
Inria MOOC but enhanced with new material on unsupervised learning, especially
clustering.

The next stages, professional and expert levels, will launch soon. We’ll also
add more courses covering other open-source libraries such as skrub (for data

> wrangling), hazardous (for survival analysis), and fairlearn (for fairness).

Reviewer: Should we link to the homepages of these libraries?

---
#### Blog Post Template ####

#### Post Information ####
title: "Skolar: an open-source initiative to democratize open data science"
date: June 30, 2025

#### Post Category and Tags ####
# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
categories:
  - Updates
tags:
  - Sponsor
  - Open Source
  - Machine Learning

#### Featured Image ####
featured-image: BSD_watermark.svg

#### Author Info ####
# Can accomodate multiple authors
# Add SQUARE Author Image to /assets/images/author_images/ folder
postauthors:
  - name: Skolar
    website: https://skolar.probabl.ai/
    image: "skolar-logo.png"
  - name: Pénélope Gittos
    website: https://www.linkedin.com/in/gittospenelope-data-analyst-growth-bilingual/
    image: "penelope_gittos.jpeg"
---
<div>
  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
  {% include postauthor.html %}
</div>

> <span style="color:red">*Sponsored blog post* </span>
> 

Reviewer 3: ```suggestion
<span style="color:red">*This blog post has been submitted by Probabl, a sponsor of scikit-learn.* </span>
```
